backend/src/main/java/org/eclipse/tractusx/semantics/hub/persistence/triplestore/SparqlQueries.java

```diff
@@ -176,12 +176,8 @@ public static Query buildFindByUrnQuery( final AspectModelUrn urn ) {
       return pss.asQuery();
    }
 
-   public static Query buildCountAspectModelsQuery( String namespaceFilter, String nameFilter, String nameType,
+   public static Query buildCountAspectModelsQuery( String namespaceFilter,
          ModelPackageStatus status ) {
-      if(StringUtils.isNotBlank(nameType) && StringUtils.isNotBlank(nameFilter)){
-         return buildExtendedSearchQuery( COUNT_ASPECT_MODELS_EXTENDED_QUERY, namespaceFilter,
```

Comment:
It seems like you can remove the query COUNT_ASPECT_MODELS_EXTENDED_QUERY as well since it is not used somewhere else.